docs/releases/v17_observer_migration.md

```diff
@@ -0,0 +1,31 @@
+
+# V17 Observer Migration Guide
+
+## Authorizations Update
+
+Observers use the `authz` module to grant authorization to execute messages with their hotkey.
+
+The following messages using `authz` have been updated:
+- `AddToOutTxTracker` to `AddOutboundTracker`
+- `AddToInTxTracker` to `AddInboundTracker`
+- `VoteOnObservedOutboundTx` to `VoteOutbound`
+- `VoteOnObservedInboundTx` to `VoteInbound`
+
+In consequence, the observers must manually add the authorization for the new messages.
+
+The authorizations can be added by interacting with the `authz` module. In this example with the `zetacored` CLI:
+
+The current authorization grants can be listed with the following command:
+```bash
+zetacored q authz grants-by-grantee [operator_address]
+```
+
+To add the authorization for the new messages, use the following command:
+```bash
+zetacored tx authz grant [grantee_address] generic --msg-type=/zetachain.zetacore.crosschain.MsgVoteInbound
+zetacored tx authz grant [grantee_address] generic --msg-type=/zetachain.zetacore.crosschain.MsgVoteOutbound
+zetacored tx authz grant [grantee_address] generic --msg-type=/zetachain.zetacore.crosschain.MsgAddOutboundTracker
+zetacored tx authz grant [grantee_address] generic --msg-type=/zetachain.zetacore.crosschain.MsgAddInboundTracker
+```
+
+Note: These new authorizations must be added before the v17 upgrade. These can be added anytime and doesn't require to restart the Zetaclient
```

Comment:
It involves adding a generic authorization in the `authz` module (the message doesn't need to be in the codebase to give the authorization). This can be done anytime even with previous releases

Reply:
The infrastructure team would have to know these in case the pipeline needs adjustment.